src/gkeClient.ts

```diff
@@ -223,6 +216,19 @@ type context = {
   name: string;
 };
 
+export type CreateKubeConfigOptions = {
```

Comment:
Doing this here to make adding additional options (like context name https://github.com/google-github-actions/get-gke-credentials/issues/86) easier